opm/autodiff/ParallelDebugOutput.hpp

```diff
@@ -106,8 +106,8 @@ namespace Opm
         virtual const WellStateFullyImplicitBlackoil& globalWellState() const { return *wellState_; }
         virtual bool isIORank () const { return true; }
         virtual bool isParallel () const { return false; }
-        virtual int numCells() const { return grid_.number_of_cells; }
-        virtual const int* globalCell() const { return grid_.global_cell; }
+        virtual int numCells() const { return Opm::AutoDiffGrid::numCells(grid_); }
```

Comment:
out of curiosity: what's the difference between `UgGridHelpers::numCells()` and `AutoDiffGrid::numCells()`?